src/schoology-extractor/data_extractor/main.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
About "data_extractor" directory name, not this file.  Python standard looks like same name for this directory as the project name.  Would suggest "schoology_extractor" since Python doesn't like dashes in import.